C_CPP/Macros in C/Macros.md

```diff
@@ -0,0 +1,118 @@
+# Macros in C
+<br>
+
+### **Something about the C preproccesor**
+Let us see what happens when we finish writing our code and send it for compilation. Just before the program gets compiled, we have something called the C preprocessor, which transforms the program before compilation. This could include including a header file, expanding a macro etc. So this is where Macros come into play. Let us see what they are. 
+<br> <br>
+
+### **What are Macros**
+A macro is basically a piece of code that is given a name. It can be defined by using the **#define** preprocessor directive. Macros are of two types: 
```

Comment:
```suggestion
A macro is a piece of code that is given a name. It can be defined by using the **#define** preprocessor directive. Macros are of two types: 
```